mbq/metrics/contrib/utils.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+DIGIT_ID_REGEX = re.compile('\/[0-9]+')
+UUID_REGEX = re.compile('\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}')
+
+
+def _sluggified_path(path):
+    return re.sub(DIGIT_ID_REGEX, '/:id', re.sub(UUID_REGEX, '/:id', path))
+
+
+def get_response_metrics_tags(status_code, path, method, content=None):
```

Comment:
Forgot to use the `content` kwargs 😄

Reply:
actually I left it out on purpose in the wsgi middleware. I just didn't think it would be useful to send content length